docs/pages/access-controls/access-lists/guide.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: A Simple Access List Example
+description: Learn how to use Access Lists to manage and audit long lived access to Teleport managed resources.
+---
+
+<Admonition type="warning">
+Access Lists are currently in Preview mode.
+</Admonition>
+
+This guide will help you set up a basic Access List.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A running Teleport cluster with some built in resources. Following the
+  [Web Application Access Guide](../../application-access/guides/connecting-apps.mdx) will create a base with a
+  simple application that we can use to demonstrate access lists.
+- An administrative user with the preset `editor` role, which will have permissions to create Access Lists.
+
+## Step 1/3. Create a test user
+
+First, we need to create a simple test user that has only the `requester` role, which has no default access
+to anything within a cluster. This user will only be used for the purposes of this guide, so you may use
+another user if you so choose.
+
+```code
+$ tctl users add test-user --roles=requester --logins=root,ubuntu,ec2-user
+```
+
+Then, follow the link provided and create credentials for this user. Try logging in to verify that no resources
+are found.
+
+## Step 2/3. Create an Access List
+
+Next, we'll create a simple access list that will grant the `access` role to its members.
+Login as the administrative user mentioned in the prerequisites. Navigate to the management pane and
+click on access lists. Click on "Create an Access List."
+
+![Navigate to create new Access List](../../../img/access-controls/access-lists/create-new-access-list.png)
+
+Here, fill in a title, description, and grant the `access` role.
+
+![Fill out Access List Fields](../../../img/access-controls/access-lists/fill-out-access-list-fields.png)
+
+Under "List Owners" select `editor` as a required role, then add your administrative user under "Add
+Eligible List Owners."
+
+![Select an owner](../../../img/access-controls/access-lists/select-owner.png)
+
+Under "Members" select `requester` as a required role, then add your test user to the access list.
+
+![Add a member](../../../img/access-controls/access-lists/add-member.png)
+
+We'll leave the the requirements fields blank and then click "Create Access List" at the bottom
+of the page.
+
+## Step 3/3. Login as the test user
+
+Again, login as the test user. When logging in now, you should now see the resources contained within
+the cluster, and should be able to interact with them as expected.
+
+## Next Steps
```

Comment:
We should add a link to Access Lists to the "next steps" for access requests guides too.